### PR TITLE
[SYCL][UR][L0] Clean up PI references in Level Zero adapter

### PR DESCRIPTION
**Pull request by jandres742**

### CODE REVIEW

File: sycl/plugins/unified_runtime/ur/adapters/level_zero/queue.hpp [hunk 1]
```diff
@@ -298,18 +300,18 @@ struct ur_queue_handle_t_ : _ur_object {
...
-  // # Get pi_event from the beginning of the cache because there are two events
-  // # there. So it is guaranteed that we do round-robin between two events -
-  // # event from the last command is appended to the cache.
-  // Operation3 = zeCommandListAppendMemoryCopy (signal ze_event1)
-  // # The same ze_event1 is used for Operation1 and Operation3.
+  // # Get ur_event_handle_t from the beginning of the cache because there are
+  // two events # there. So it is guaranteed that we do round-robin between two
+  // events - # event from the last command is appended to the cache. Operation3
+  // = zeCommandListAppendMemoryCopy (signal ze_event1) # The same ze_event1 is
+  // used for Operation1 and Operation3.
```
Comment (smaslov-intel) [inline sycl/plugins/unified_runtime/ur/adapters/level_zero/queue.hpp:314]: NIT: 
```suggestion
  // # events - event from the last command is appended to the cache.
  // Operation3 = zeCommandListAppendMemoryCopy (signal ze_event1)
  // # The same ze_event1 is used for Operation1 and Operation3.
```

File: sycl/plugins/unified_runtime/ur/adapters/level_zero/queue.hpp [hunk 2]
```diff
@@ -513,14 +515,13 @@ struct ur_queue_handle_t_ : _ur_object {
...
-// \param CommandType various command type determined by the caller
-// \param CommandList is the command list where the event is added
-// \param IsInternal tells if the event is internal, i.e. visible in the L0
+// This helper function creates a ur_event_handle_t and associate a
+// ur_queue_handle_t. Note that the caller of this function must have acquired
+// lock on the Queue that is passed in. \param Queue ur_queue_handle_t to
+// associate with a new event. \param Event a pointer to hold the newly created
+// ur_event_handle_t \param CommandType various command type determined by the
+// caller \param CommandList is the command list where the event is added \param
+// IsInternal tells if the event is internal, i.e. visible in the L0
```
Comment (smaslov-intel) [inline sycl/plugins/unified_runtime/ur/adapters/level_zero/queue.hpp:524]: slipped formatting of params

**Review by smaslov-intel — APPROVED** [review]
LGTM. Just a few formatting suggestions

File: sycl/plugins/unified_runtime/ur/adapters/level_zero/queue.hpp (hunk 1 above)
Comment (jandres742) [reply sycl/plugins/unified_runtime/ur/adapters/level_zero/queue.hpp:314]: thanks @smaslov-intel. Changed.

File: sycl/plugins/unified_runtime/ur/adapters/level_zero/queue.hpp (hunk 2 above)
Comment (jandres742) [reply sycl/plugins/unified_runtime/ur/adapters/level_zero/queue.hpp:524]: thanks @smaslov-intel. Addressed.

**Review by kbenzie — APPROVED** [review]
nit: `Ur` seems odd when its an acronym, `UR` would make more sense to me.

**Comment by kbenzie** [comment]
> > nit: `Ur` seems odd when its an acronym, `UR` would make more sense to me.
> 
> thanks @kbenzie . Changed it. Please see latest commit.

looks good 👍

**Comment by jandres742** [comment]
@intel/llvm-gatekeepers please merge when possible

**Pull request merged**